test/Project.toml

```diff
@@ -0,0 +1,7 @@
+[deps]
+GroupsCore = "d5909c97-4eac-4ecc-a3dc-fdd0858a4120"
```

Comment:
I don't mind this, but technically it should also be possible to do that you did for e.g. `Random` and modify the tests to replace `GroupsCore` by `AbstractAlgebra.GroupsCore`, I think?

Reply:
I was trying to get some working version with succeeding CI for the first changes. There are some more, like `GroupsCore`, I want to tackle in a second step.

Reply:
This is kind of tricky due to
https://github.com/kalmarek/GroupsCore.jl/blob/f90a9fee9c9b6182efe40d453982732f66ab4a27/test/conformance_test.jl#L1

in `GroupsCore.jl`, which is included in the tests here
https://github.com/Nemocas/AbstractAlgebra.jl/blob/82826052be6beee1d8d153fb7266088b997d85e7/test/generic/PermGroupAPI-test.jl#L5

 I see three ways to proceed:

1. Delete this line in `GroupsCore.jl/test/conformance_test.jl`.
2. Do some ugly hacking to include everything but the first line of that file.
3. Leave `GroupsCore` as a test dependency of AA.

Which of these would you prefer?

Reply:
For now just do 3 so we can proceed